docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml

```diff
@@ -589,33 +597,211 @@ depending on where they wind up.-->
             </mediaobject>
           </figure></para>
 
-        <para>The package maps page, displays all the package maps loaded on
+        <para>The package maps page displays all the package maps loaded on
         your cluster. You can Add, Activate, Deactivate, Delete, or Open a
         package map. To examine a package map, select a package map from the
         list.</para>
 
-        <para>You can select the <emphasis role="bold">Validate Package
-        Map</emphasis> tab to validate a package map.</para>
+        <para>To update the package maps you are using, you would either edit
+        the package map file or add a new one and then activate it. You could
+        later delete the old one.</para>
 
-        <para><figure>
-            <title>Validate Package Maps</title>
+        <sect3>
+          <title id="Packagemap_Actions">Package Map Actions</title>
 
-            <mediaobject>
-              <imageobject>
-                <imagedata fileref="../../images/ECLWaPkMgr.jpg" />
-              </imageobject>
-            </mediaobject>
-          </figure></para>
+          <para>You can perform actions on your package maps from the Package
+          Maps tab in ECL Watch.</para>
+
+          <para><emphasis role="bold">Package Map Open</emphasis></para>
+
+          <para>To examine a package map, select the package map and press the
+          <emphasis role="bold">Open</emphasis> action button. This opens a
+          new tab where you can see the package parts.</para>
```

Comment:
The term parts here is a bit ambiguous.  Opening the packagemap creates a tab for that packagemap and from there you have access to child tabs for info, xml, validate and now "parts".